app/src/main/java/com/kickstarter/ui/intentmappers/IntentMapper.java

```diff
@@ -24,4 +24,14 @@
 
     return Uri.parse(string);
   }
+
+  public static boolean intentFromAppBanner(final @NonNull Intent intent) {
```

Comment:
The method name makes this read like it's returning an intent from a banner – might consider something like `appBannerIsSet`